addons/mail/migrations/12.0.1.0/end-migration.py

```diff
@@ -64,7 +64,16 @@ def fill_mail_thread_message_main_attachment_id(env):
         )
 
 
+def remove_activity_date_deadline_column(env):
+    activity_mixin_models = [k for k in env.registry if issubclass(
+        type(env[k]), type(env['mail.activity.mixin'])) and env[k]._auto]
+    _column_renames = {model._table: ('activity_date_deadline', None)
```

Comment:
```suggestion
    _column_renames = {env[model]._table: ('activity_date_deadline', None)
```

Reply:
`model` is not a string. `model` is the model class.